src/styles/blocks/stars.css

```diff
@@ -0,0 +1,56 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  list-style: none;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  position: relative;
+  top: 2px;
+  padding: 0 2px;
+  margin-left: 2px;
+}
+
+.stars--1 span:first-child::before {
+  content: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 0;
+  top: -2px;
+}
+
+.stars--2 span:nth-child(-n + 2)::before {
+  content: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 0;
+  top: -2px;
+}
+
+.stars--3 span:nth-child(-n + 3)::before {
+  content: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 0;
+  top: -2px;
+}
+
+.stars--4 span:nth-child(-n + 4)::before {
+  content: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 0;
+  top: -2px;
+}
+
+.stars--5 span:nth-child(-n + 5)::before {
+  content: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  position: absolute;
```

Comment:
as you can see here, you're repeating quite a bit of styles, could you please try to figure out a better way to do it? (you can use `,` to use same styles for more than 1 selector 😉 )

Reply:
of course, all fixed now thanks